lib/pipedrive/operations/read.rb

```diff
@@ -24,6 +24,23 @@ def chunk(params = {})
         res
       end
 
+      def each_items(id, item_path_name, params = {}, before_request = nil, &block)
```

Comment:
We might need to add before_request params to each method as well. Since we need to use it to pull leads and also want to rate limit it.